plugins/inventory/inventory.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+name: inventory
+author:
+  - Polona Mihalič (@PolonaM)
+short_description: Inventory source from Terraform state file.
+description:
+  - Builds an inventory from specified state file.
+  - Does not support caching.
+version_added: 1.0.0
```

Comment:
Not added in 1.0.0, since that was already released :) Use 1.1.0 instead please, for lack of a better version.

Reply:
updated